app/components/Carousel/NavButtonStories/index.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import { StyledBtn } from './NavButton.styles';
+import Icon from 'components/Icon';
+
+const NavButton = ({ handleClick, type, ...props }) => {
+  return (
+    <>
+      <StyledBtn onClick={handleClick} {...props}>
+        <Icon type={type} color="#fff" />
+      </StyledBtn>
+
+      <StyledBtn onClick={handleClick} {...props}>
+        <Icon type={type} color="#fff" />
+      </StyledBtn>
+    </>
+  );
+};
+
+NavButton.propTypes = {
+  handleClick: PropTypes.func,
+  backgroundColor: PropTypes.string,
+  type: PropTypes.oneOfType[('faChevronLeft', 'faChervonRight')],
+};
```

Comment:
```suggestion
NavButton.propTypes = {
  handleClick: PropTypes.func,
  backgroundColor: PropTypes.string,
  type: PropTypes.oneOf(['chevron-left', 'chevron-right']),
};
```
Changes in line 22 enables checkbox for left and right chevrons in Storybook.

Reply:
nice one, this one is better